pipeline/transform.py

```diff
@@ -0,0 +1,81 @@
+"""Script for transforming the data extracted from the api and web-scraping."""
+
+from datetime import datetime, timezone
+from extract import insert_protocol_url, get_stem_url, get_sales_data
+
+
+def convert_unix_to_datetime(unix_timestamp: float) -> str:
+    """Constructs and returns a string representing date and time from a Unix timestamp."""
+
+    if not isinstance(unix_timestamp, float):
+        raise TypeError(
+            f"Expected a float for unix_timestamp, but got {
+                type(unix_timestamp)}"
+        )
+
+    if unix_timestamp < 0:
+        raise ValueError("Unix timestamp cannot be negative.")
+
+    unix_timestamp_seconds = int(unix_timestamp)
+
+    date_time = datetime.fromtimestamp(unix_timestamp_seconds, tz=timezone.utc)
+
+    utc_date_time_now = datetime.now(timezone.utc)
+
+    if date_time > utc_date_time_now:
+        raise ValueError(
+            f"The provided Unix timestamp corresponds to a future date or time: {
+                date_time}"
+        )
+    return date_time.strftime("%Y-%m-%d %H:%M:%S")
+
+
+def clean_data(sales_data: dict) -> dict:
+    """Takes the sales data as a dict, and removes specific, unnecessary keys."""
+
+    if not isinstance(sales_data, dict):
+        raise TypeError("Input data must be a dictionary.")
+
+    keys_to_remove = [
```

Comment:
Could add this as a global variable

Reply:
Added in [d8a3aa9](https://github.com/Zhi-704/c11-apollo-bandcamp-tracker/pull/33/commits/d8a3aa9d48ffa9ae0af83619f43ccb598847620b)